### ISSUE TITLE
SSL certificate lookup should always try lower cased hostnames for let's encrypt

### ISSUE THREAD

**Issue by FirefighterBlu3** [apple/cups#4919]
CUPS inconsistently uses case sensitive lookups based on the hostname. Let's Encrypt uses lower cased hostnames.

```
~$ hostname
Scott.Blue-Labs.org

[pid 18210] access("/etc/letsencrypt/live/scott.blue-labs.org/Scott.Blue-Labs.org.crt", R_OK) = -1 ENOENT (No such file or directory)
[pid 18210] access("/etc/letsencrypt/live/Scott.Blue-Labs.org/fullchain.pem", R_OK) = -1 ENOENT (No such file or directory)
[pid 18210] access("/etc/letsencrypt/live/Blue-Labs.org/fullchain.pem", R_OK) = -1 ENOENT (No such file or directory)
[pid 18210] access("/etc/letsencrypt/live/scott.blue-labs.org/Scott.Blue-Labs.org.crt", R_OK) = -1 ENOENT (No such file or directory)
[pid 18210] open("/etc/letsencrypt/live/scott.blue-labs.org/Scott.Blue-Labs.org.key", O_RDONLY) = -1 ENOENT (No such file or directory)
```

the real fullchain.pem file does exist, with a lower cased directory.

`/etc/letsencrypt/live/scott.blue-labs.org/fullchain.pem`

**Comment by michaelrsweet** [comment]
Sigh... OK, scheduling fix for 2.2.2.

**Comment by michaelrsweet** [comment]
[master bd31658] Let's Encrypt certificates did not work when the hostname contained uppercase letters (Issue #4919)
